dimensiondata/src/main/java/org/jclouds/dimensiondata/cloudcontrol/DimensionDataCloudControlApi.java

```diff
@@ -45,4 +46,7 @@
 
    @Delegate
    TagApi getTagApi();
+
+   @Delegate
+   CustomerImageApi getCustomerImageApi();
```

Comment:
A new API added as part of this so that we can delete the image we create when we test the clone api.